### DISCUSSION TITLE
What is the location/s of the configuration files on MacOS?

### DISCUSSION THREAD

**Discussion by cecobask** [MonitorControl/MonitorControl#1682] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

I installed the app via the `.dmg`file provided in the releases section and have since tweaked the settings.
I would like to migrate my settings to a homebrew installation of the app in order to automate installation via a `Brewfile`.
What is the location/s of configuration files, where my settings are saved, on MacOS?

**Comment by waydabber** [comment, marked as answer]
It should be under `~/Library/Preferences/app.monitorcontrol.MonitorControl.plist`

**Reply by cecobask** [reply]
That’s correct, the settings were there. Is this file suitable to commit into version control for backup? I would normally restore configurations on new machines and expect the app to work in the same way.
